packages/skeleton/src/lib/stores.ts

```diff
@@ -0,0 +1,22 @@
+import { browser } from '$app/environment';
+import { readable } from 'svelte/store';
+
+/** Prefers reduced motion  */
+const reducedMotionQuery = '(prefers-reduced-motion: reduce)';
+const getInitialMotionPreference = () => {
+	if (!browser) return false;
+	return window.matchMedia(reducedMotionQuery).matches;
+};
+export const prefersReducedMotion = readable(getInitialMotionPreference(), (set) => {
```

Comment:
@Mahmoud-zino @ryceg per this store - correct me if I'm wrong, but wasn't it determined that this is something that will need to be enforced within each transition itself? While CSS transitions will support it automatically do to `prefers-reduced-motion`?

I'm worried that we're trying to tackle too many things at once here. Custom transitions and their typing should be the first consideration, then we can circle back to a11y strategies. Basically make it work, then make it work better.

Thoughts?